docker/jaeger/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM jaegertracing/all-in-one:1.9
```

Comment:
do we need this file? as in, can we just refer to the upstream image in the docker-compose?

Reply:
Maybe.. let me test.